### DISCUSSION TITLE
Setting Custom Host Names

### DISCUSSION THREAD

**Discussion by jwcooper** [community/community#25835] [Actions]
<p>Is it possible to use something like the docker-compose extra_hosts property or even the travis’ ‘addons -&gt; hosts’ setup (<a href="https://docs.travis-ci.com/user/hosts/" rel="nofollow noopener">https://docs.travis-ci.com/user/hosts/</a>) which updates the /etc/resolv to set custom host names?</p>
<p>It would be nice be able to use something like ‘redis’ rather than always having to use 127.0.0.1 to simplify our existing configuration.</p>

**Comment by chrispat** [comment, marked as answer]
<p>There is nothing in the YAML but you should be able to script updating that file as all jobs run with passwordless sudo.</p>

**Comment by jwcooper** [comment]
<p>Thank you! That seems quite obvious now after your response. :wink:</p>
